docs/manual/docs/administrator-guide/configuring-the-catalog/user-interface-configuration.md

```diff
@@ -177,7 +177,6 @@ This section is for configuring the map shown when viewing a record.
     -   **wmts**: generic WMTS layer, required properties: `name, url`.
     -   **tms**: generic TMS layer, required property: `url`.
     -   **osm**: OpenStreetMap default layer, no other property required.
-    -   **stamen**: Stamen layers, required property: `name`.
```

Comment:
Should we be adding *stadiamaps* here at the same time we remove *stamen* ?

Reply:
The version of OpenLayers used in GeoNetwork doesn't support StadiaMaps. We can upgrade OpenLayers, there is a pull request on-going https://github.com/geonetwork/core-geonetwork/pull/7421, but not sure about the status and if it's even possible with the GeoNetwork code to do such upgrade.